Traceback (most recent call last):
  File "/home/rgg2706/Multimodal-Sentiment-Analysis/Models/MultimodalOpinionAnalysis/run_project.py", line 237, in <module>
    run_sentiment_classifier(multimodal_train_loader, multimodal_val_loader, multimodal_test_loader, params, device)
  File "/home/rgg2706/Multimodal-Sentiment-Analysis/Models/MultimodalOpinionAnalysis/run_project.py", line 204, in run_sentiment_classifier
    train(train_loader, validation_loader, test_loader, params, device)
  File "/home/rgg2706/Multimodal-Sentiment-Analysis/Models/MultimodalOpinionAnalysis/run_project.py", line 164, in train
    loss = F.cross_entropy(logits, labels.long(), weight=class_weights)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/.autofs/tools/spack/var/spack/environments/default-nlp-x86_64-24072401/.spack-env/._view/wtxwc3mlkmzy7fbaxlum2674jarpitc2/lib/python3.11/site-packages/torch/nn/functional.py", line 3053, in cross_entropy
    return torch._C._nn.cross_entropy_loss(input, target, weight, _Reduction.get_enum(reduction), ignore_index, label_smoothing)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
RuntimeError: Expected all tensors to be on the same device, but found at least two devices, cuda:0 and cpu! (when checking argument for argument target in method wrapper_CUDA_nll_loss_forward)
